src/gluonts/mx/distribution/binned.py

```diff
@@ -70,6 +70,22 @@ def __init__(
     def F(self):
         return getF(self.bin_log_probs)
 
+    @property
+    def support(self) -> Tuple[Tensor, Tensor]:
+        F = self.F
+        return (
+            F.broadcast_minimum(
```

Comment:
Naively, I would have thought this is `[min(bin_edges), max(bin_edges)]`. 
Is this due to this last catch-all bin?

Reply:
Yes exactly.

Reply:
If the mixture approach works well, we could maybe simplify the binned distribution to have finite support and remove this catch-all bin.

Reply:
But then the binned would not be standalone... probably we will need to add this as an option.